config/gregtech/worldgen/overworld/basaltic_mineral_vein.json

```diff
@@ -1,35 +1,40 @@
 {
-  "weight": 80,
-  "density": 0.5,
-  "max_height": 80,
-  "min_height": 50,
-
-  "generator": {
-    "type": "ellipsoid",
-    "radius": [15, 14]
-  },
-  "filler": {
-    "type": "simple",
-    "value": {
-      "type": "weight_random",
-      "values": [
-        {
-          "weight": 30,
-          "value": "ore:basaltic_mineral_sand"
-        },
-        {
-          "weight": 30,
-          "value": "ore:granitic_mineral_sand"
-        },
-        {
-          "weight": 20,
-          "value": "ore:fullers_earth"
-        },
-        {
-          "weight": 20,
-          "value": "ore:gypsum"
+    "weight": 80,
+    "density": 0.5,
+    "max_height": 80,
+    "min_height": 50,
+    "generator": {
+        "type": "ellipsoid",
+        "radius": [
+            15,
+            14
+        ]
+    },
+    "filler": {
+        "type": "simple",
+        "value": {
+            "type": "weight_random",
+            "values": [
+                {
+                    "weight": 30,
+                    "value": "ore:basaltic_mineral_sand"
+                },
+                {
+                    "weight": 30,
+                    "value": "ore:granitic_mineral_sand"
+                },
+                {
+                    "weight": 20,
+                    "value": "ore:fullers_earth"
```

Comment:
I think we should just get rid of Fuller's Earth. It is probably one of the most useless ores, with it's only good byproduct being Magnesium, which has tons of other free sources.

Reply:
should i yeet the whole vein?

Reply:
nvm i wont